common/src/dodeca.rs

```diff
@@ -24,48 +34,57 @@ pub enum Side {
 }
 
 impl Side {
-    #[inline]
-    pub fn from_index(x: usize) -> Self {
-        use Side::*;
-        const VALUES: [Side; SIDE_COUNT] = [A, B, C, D, E, F, G, H, I, J, K, L];
-        VALUES[x]
-    }
+    pub const COUNT: usize = 12;
```

Comment:
Why not `VALUES.len()`? Similar for `VERTEX`.

Reply:
Good question. I probably should have done the same for `common::world::Material` earlier, which I modeled this on.

Reply:
tbf `len` might not have been `const` back then!